tests/lib/rules/prefer-explicit-assert.test.ts

```diff
@@ -6,86 +6,96 @@ const ruleTester = createRuleTester();
 
```

Comment:
Let me know if I added enough tests as I'm not totally confident with the latest changes and what it implies in the different rules 🙂

Reply:
The tests variants are more than fine! I would suggest to include a custom query method for testing this rule. You only need to create `const COMBINED_QUERIES_METHODS = [...ALL_QUERIES_METHODS, 'ByIcon']` and use this array of methods rather than `ALL_QUERIES_METHODS`